README.md

```diff
@@ -175,6 +176,7 @@ nexusIQScan {
     modulesExcluded = listOf("module-1", "module-2") // Optional. For multi-module projects, the names of the sub-modules to exclude from scanning and evaluation.
     dirExcludes = "some-ant-pattern" // Optional. Comma separated ant-like glob patterns to select directories/archives that should be excluded. For Android projects we suggest using "**/classes.jar,**/annotations.zip,**/lint.jar,**/internal_impl-*.jar"
     dirIncludes = "some-ant-pattern" // Optional. Comma separated ant-like glob patterns to select directories/archives that should be examined
+    scanFolderPath = "/path/to/project/root" // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the working directory (where gradle is executed).
```

Comment:
```suggestion
    scanFolderPath = "/path/to/project/root" // Optional. Path to the project root, under which the '.git' folder usually resides. Defaults to the project's root directory.
```

Reply:
This does not reflect my suggestion (see comment above).

Reply:
I don't think we need to document it at all. The project's root directory is what we need. Users can only mess it up.

Reply:
Agreed!